stylix/testbed.nix

```diff
@@ -149,8 +149,8 @@ let
   # This generates a copy of each testbed for each of the following themes.
   makeTestbeds =
     testbed:
-    map (makeTestbed testbed) [
-      {
+    lib.mapAttrsToList (makeTestbed testbed) {
```

Comment:
This `map` would need to be updated to handle attrs (with `description`) and lists (without `description`) differently.